doc/man3/OSSL_PARAM_TYPE.pod

```diff
@@ -449,6 +449,36 @@ or, more succinctly, like this:
     OSSL_PARAM_optional_set_utf8_string(params, "bar", "bar value");
     OSSL_PARAM_optional_set_utf8_string(params, "cookie", "cookie value");
 
+=head2 Example 3
+
+This example is of sending RSA public key parameters to a function.
+To send a public key:
+
+    OSSL_PARAM params[3];
+    size_t n = 0;
+    unsigned char n_buf[512];
+
+    BN_bn2nativepad(rsa_n, n_buf, sizeof(n_buf));
+
+    params[n++] = OSSL_PARAM_construct_BN("n", n_buf, sizeof(n_buf), NULL);
+    params[n++] = OSSL_PARAM_construct_uint32("e", rsa_e, NULL);
+    params[n] = OSSL_PARAM_END;
```

Comment:
Heh, have you tested that this line goes through a compiler?

Reply:
I wondered if anyone would notice :)